Khan/wonder-blocks
# Update Storybook to v6.4

## Summary:

This will enable us to use Storybook in the near future with:

- Interactive tests
- Faster tooling (on-demand architecture)
- ESLint support for writing stories using best practices/recommendations.
- Feature flags (?) why not!

Issue: XXX-XXXX

## Test plan:

- Run `yarn install -W && yarn start:storybook"

- Verify that the SB environment continues working without problems.
